public/locales/en-US.properties

```diff
@@ -77,3 +70,31 @@ unlocking-wallet = Unlocks..
 adding-account = Adds new account..
 loading-refresh = Loads transactions..
 sending-funds = Sending funds..
+
+# Onboard
+onboard-button-back = Back
```

Comment:
Do we want to use a prefix for everything? Like onboard__ and create__ or something similar?

Reply:
I think it can be smart so that we know the context of the text. However, when we finish with most of the text it's probably easier to see what are the common things, and what we might need context for (if any).